src/compiler/python_generator.cc

```diff
@@ -232,10 +232,13 @@ static void PrintAllComments(const DescriptorType* desc, Printer* printer) {
 bool PrintBetaServicer(const ServiceDescriptor* service,
                        Printer* out) {
   out->Print("\n\n");
-  out->Print("class Beta$Service$Servicer(object):\n", "Service",
-             service->name());
+  out->Print("class Beta$Service$Servicer(object):\n",
+   "Service", service->name());
   {
     IndentScope raii_class_indent(out);
+    out->Print("\"\"\"The Beta API is deprecated for 0.15.0 and later. It is recommended\n"
```

Comment:
Let's make these more compliant with [the controlling style guide](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments) - blank line between short summary string and remaining text, and put a period at the end f the last sentence.

When you look at the generated code, how does it turn out?